src/BuildScriptGenerator/DefaultDockerfileGenerator.cs

```diff
@@ -255,5 +261,49 @@ private string ConvertToRuntimeVersion(string platformName, string platformVersi
         {
             return this.platformDetector.GetCompatiblePlatforms(ctx);
         }
+
+        private void ValidateDockerfileImages(DockerfileProperties dockerfileProperties)
+        {
+            if (string.IsNullOrEmpty(dockerfileProperties.BuildImageName))
+            {
+                this.LogInvalidDockerfileImageException(
+                    "Provided build image name parsed from the provided --build-image argument is empty. " +
```

Comment:
Nit: should we mention something about the format expected? I imagine that it's kind of implied from the `cli:stable` example, but I don't know if some customers could be confused about this. If we have a meaningful comment in the help of the --build-image argument, please feel free to ignore.

Reply:
The help for the `--build-image` argument is currently the following:

> _The mcr.microsoft.com/oryx build image to use in the dockerfile, provided in the format '<image>:<tag>'. If no value is provided, the 'cli:stable' Oryx image will be used._

I'll skip this comment since the documentation is already in place to specify the format, but I don't mind calling it out again in the exception message if necessary.

Reply:
OK sounds good, thanks!